Source/Web/SvgHandler.cs

```diff
@@ -1,3 +1,4 @@
+#if __EXCLUDED__

```

Comment:
so, `__EXCLUDED__` actually means "Web included", or did I misunderstand this? Shouldn't the web part be included for .NET implementations?

Reply:
Again, me being lazy. There are a number of components that would not compile - and it was easier to remove the whole.
In retrospect, I should leave it enabled for the version of dotnet that support it. I'll review it tomorrow.

Reply:
Ok, thanks!